src/main/java/frc/robot/RobotMap.java

```diff
@@ -32,6 +32,9 @@ public final class RobotMap {
   /** Lift grabbing pneumatic channels */
   public static final int LIFT_GRABBING_FORWARD_CHANNEL = 1, LIFT_GRABBING_REVERSE_CHANNEL = 2;
 
+  /** Skid pneumatic channels */
+  public static final int SKID_FORWARD_CHANNEL = 3, SKID_REVERSE_CHANNEL = 4;
```

Comment:
Let's double check, there may also be multiple skid pneumatic cylinders? We can probably just leave off all the skid details since it's so uncertain right now tho